.ado/jobs/project-reunion.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - ${{ if eq(config.BuildEnvironment, parameters.buildEnvironment) }}:
       - ${{ each matrix in config.Matrix }}:
         - job: Reunion${{ matrix.Name }}
-          displayName: Project Reunion ${{ matrix.Name }}
+          displayName: Windows App SDK ${{ matrix.Name }}
```

Comment:
Could we rename this file from `project-reunion`?

Reply:
yep, done.